web/static/js/components/user_list_item.jsx

```diff
@@ -8,13 +8,22 @@ function UserListItem(props) {
   return (
     <li className={`item ${styles.wrapper}`}>
       <div className="ui center aligned grid">
-        <div className="ui row">{ userName }</div>
-        <i className="huge user icon" />
+         {IconTag(props.user)}
+         <div className="ui row">{ userName }</div>
       </div>
     </li>
   )
 }
 
+const IconTag = function(user) {
```

Comment:
this is a great change. thanks for taking it on unprompted. only blocker for merging is the need for a couple unit tests to the `UserListItem` component?